stream-chat-android-sample/src/main/kotlin/io/getstream/chat/sample/feature/channels/ChannelsFragment.kt

```diff
@@ -22,7 +24,15 @@ import io.getstream.chat.sample.databinding.FragmentChannelsBinding
 
 class ChannelsFragment : Fragment() {
 
-    private val channelsViewModel: ChannelsViewModel by viewModels { ChannelsViewModelFactory() }
+    private val channelsViewModel: ChannelsViewModel by viewModels {
+        ChannelsViewModelFactory(
+            filter = Filters.and(
+                Filters.eq("type", "messaging"),
+                Filters.`in`("members", listOf(ChatClient.instance().getCurrentUser()?.id ?: "")),
+                Filters.or(Filters.notExists("draft"), Filters.eq("draft", false)),
+            ),
+        )
+    }
```

Comment:
Even though we don't create "draft" channels in the old sample app we still share the same environment with the new sample app where such channels are a common occurrence.